src/main/java/com/mraof/minestuck/item/BarbasolBombItem.java

```diff
@@ -23,24 +24,29 @@ public BarbasolBombItem(Properties properties)
     public ActionResult<ItemStack> use(World worldIn, PlayerEntity playerIn, Hand handIn)
     {
         ItemStack item = playerIn.getItemInHand(handIn);
-        
-        if(!playerIn.isCreative())
+    
+        if(!CreativeShockEffect.doesCreativeShockLimit(playerIn, 0))
```

Comment:
I don't think this item should check for creative shock as it already prevents block breakage if `playerIn.abilities.mayBuild` is false.